server/startup/jedis.js

```diff
@@ -37,8 +44,104 @@ Meteor.startup( function() {
 	//Accounts.config( {forbidClientAccountCreation:true});
 });
 
+var addUsersToLocation = function() {
+	var owner = Meteor.users.findOne({_id:'testadmin'});
+	var rawUsers = Meteor.users.rawCollection();
+	var pipeline = [{'$group': {_id:'$profile.location', usernames: {$push : "$username"}}}]
+	Meteor.wrapAsync( rawUsers.aggregate, rawUsers)(pipeline, function(err, usersByLocation) {
+		if( err ) {
+			console.log( err )
+		} else {
+			usersByLocation.forEach(function(location) {
+				var channel = ChatRoom.findOne({name:location._id, t:'c'});
+				if( channel ) {
+					// location exists so add user if they don't already belong
+					addUsersToRoom(location.usernames, channel._id, false)
+				} else {
+					// location doesn't exist so create new channel with users
+					createChannel(owner, location._id, location.usernames);
+				}
+			})
+		}
+
+	})
+}
+
+var addUserToLocationChannel = function(user) {
+	var channel;
+	var chId;
+	if( _.isUndefined(user.profile) || _.isUndefined(user.profile.location) ) {
```

Comment:
Just the console.  The next step would be to email an admin that their user data is incomplete.  That is not our concern.